.github/workflows/.windows.yml

```diff
@@ -360,10 +360,14 @@ jobs:
         name: Starting test daemon
         run: |
           Write-Host "Creating service"
-          If ("${{ matrix.runtime }}" -eq "containerd") {
+          If ("${{ matrix.runtime }}" -eq "containerd" -and "${{ matrix.os }}" -eq "windows-2019") {
             $runtimeArg="--containerd=\\.\pipe\containerd-containerd"
             echo "DOCKER_WINDOWS_CONTAINERD_RUNTIME=1" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
           }
+          If ("${{ matrix.runtime }}" -eq "containerd" -and "${{ matrix.os }}" -eq "windows-2022") {
```

Comment:
Because in that case, this probably would never be met (`windows-2022`)?


(maybe I'm wrong, but .. my eye fell on the `windows-2019` added above 😅)

Reply:
@thaJeztah these are just copy paste from my couple of year's old PR #43893
On that time we had both containerd and non-containerd workflows for both Windows Server 2019 and 2022 so my thinking was to use external containerd in 2019 and Docker managed in 2022.

But now when PRs does not trigger 2019 builds it might be better to use just engine managed version for both.

Reply:
Resolving this as well since this step to "Start Containerd" manually has been removed in favour of the Docker-managed instance.